spec/tSpec.js

```diff
@@ -64,6 +64,14 @@ describe('Ember.I18n.t', function() {
     expect(Ember.I18n.t('nothing.here')).to.equal('there.is.nothing.here.to.see');
   });
 
+  it('warns with a custom message that includes the passed context', function() {
+    Ember.I18n.missingMessage = function(key, context) {
```

Comment:
Good catch, I didn't think of it since I just copied the above test, which also doesn't reset it. I'll fix that one while I'm in there.